UPGRADE-3.4.md

```diff
@@ -68,6 +68,9 @@ Process
 
  * The `Symfony\Component\Process\ProcessBuilder` class has been deprecated,
    use the `Symfony\Component\Process\Process` class directly instead.
+   
+ * The `Symfony\Component\Process\Process` __construct `cwd` argument will throw 
+   a deprecation warning on `run` if the folder does not exist.
```

Comment:
Alternatively, I'm going away for the weekend. If somebody else wants to apply these as a patch to 3.4 instead of master, i'm more than happy for you to go ahead and do it.